content/products/desktop/index.html

```diff
@@ -0,0 +1,109 @@
+images: []
+aliases: ["/[products/cockpit/]"]
+---
+<section class="section section-sm bg-gradient-dark ">
+    <div class="position-absolute w-100 section pb-7 pt-7 d-none d-md-block">
+        <div class="container">
+        <div class="row justify-content-center align-items-center">
+            <div class="col-3">
+            <img
+                class="w-100"
+                style="transform: scale(1.75); opacity: .9;"
+                src="/images/heroes/cloud.svg"
+            />
+            </div>
+            <div class="col-1">
+            </div>
+            <div class="col-2"></div>
+            <div class="col-1">
+            </div>
+            <div class="col-3">
+            <img
+                class="w-100"
+                style="transform: scaleX(-1) scale(1.2)"
+                src="/images/heroes/cloud.svg"
+            />
+            </div>
+        </div>
+        </div>
+    </div>
+    <div class="container pb-7 pt-7 mb-4">
+        <div class="row justify-content-center align-items-center text-white">
+            <h1 class="text-center m-0 pt-5">LocalStack Desktop</h1>
+            <p class="text-center mt-4" style="font-size: 1.5rem">
+                LocalStack Desktop, a fully-integrated desktop experience for LocalStack.
+                <br/>
+                Start using LocalStack - without the fuss
+            </p>
+        </div>
+        <div class="row justify-content-center text-center">
+            <div class="col-12 d-flex justify-content-center gap-3">
+                <a href="https://docs.localstack.cloud/user-guide/tools/localstack-desktop/" style="border: 1px solid white" class="btn btn-second text-white btn-lg">
+                    LEARN MORE
+                </a>
+                <a href="https://app.localstack.cloud/desktop-download/" class="btn btn-primary btn-tertiary btn-lg">
+                    DOWNLOAD
+                </a>
+            </div>
+        </div> 
+    </div>
+</section> 
+<section class="section section-md pb-17">
+    <div class="container overlay-card mt-n8">
+        <div class="row mb-5 mt-5">
+            <div class="col-12 col-md-7 col-xl-6 order-1 order-md-0">
+                <img src="mainpage.png" class="img-fluid" />
+            </div>
+            <div class="col-12 col-md-5 col-xl-6 order-0 order-md-1">
+                <h2 class="h1">Control LocalStack</h2>
+                <ul>
+                    <li>Start and Stop LocalStack instances with the click of a button</li>
+                    <li>See the current status of your LocalStack instances</li>
+                    <li>Easily access the application via a tray icon</li>
+                </ul>
+            </div>
+        </div>
+    </div>
+</section> 
+<section class="section section-md bg-gradient-dark">
+    <div class="container overlay-card mt-n15">
+        <div class="row mb-5">
+            <div class="col-12 col-md-5 col-xl-6">
+                <h2 class="h1">Instant insights</h2>
```

Comment:
Could be cool to do a fancy slider to show both pages but I'll leave it for somebody else in case :joy:

Reply:
Done